### ISSUE TITLE
HUD not popping up when using AstroVim

### ISSUE THREAD

**Issue by jaikb** [Olical/conjure#565]
So I recently switched from NvChad to AstroVim and I installed Conjure and everything works fine, but I can't seem to get the HUD to pop up now when I run commands. The logs and stuff run fine, but no idea why the HUD won't pop up. Any ideas why or how to fix?

**Comment by jaikb** [comment]
I solved this issue by adding custom configs to set display HUD to true in my `users.lua`

**Comment by practicalli-johnny** [comment]
I assume this is using the [AstroCommunity Clojure pack](https://github.com/AstroNvim/astrocommunity/tree/main/lua/astrocommunity/pack/clojure) that I contributed to that project.

I've never shown the REPL log/prompt when using any Clojure editor as inline results were found to be more effective. I made the opinionated choice to set the HUD to false when connecting to a REPL process.  

I prefer to use `,lt` to open the REPL log in a separate tab, using `gt` to toggle between source code and REPL log.

I will be adding more infomation to the AstroCommunity Community Pack to make these choices clearer.  I am also about to refactor the plugins and configurations to simplify the pack and remove some of the choices I made (although still hide the HUD).

If anyone strongly disagres with the choices I make, then let me know and I will consider moving them into the [practicalli/astronvim-user-config](https://github.com/practicalli/astronvim-user-config) repository instead
